topo_processor/cli/validate.py

```diff
@@ -34,7 +34,7 @@
     is_flag=True,
     help="Use verbose to display trace logs (it might be slower).",
 )
-def main(item: str, collection: str, metadata: str, verbose: str) -> None:
+def main(item: bool, collection: bool, metadata: str, verbose: str) -> None:
```

Comment:
if these are `bool` what exactly does `if item == collection` mean on line 49?

Reply:
There might be a prettier way to do that.
That means we call `validate_stac(metadata)` - either with `Item` and `Collection` validation if the user hasn't specify any of them or had specified `True` for both of them.
If one of the flag `Collection` or `Item` has been specified, we call `validate_stac` with those flags regardless which one is `True` or `False`